tests/test_open_images_format.py

```diff
@@ -8,12 +8,82 @@
 
 import numpy as np
 
-from datumaro.plugins.open_images_format import OpenImagesImporter
-from datumaro.util.test_utils import compare_datasets_strict
+from datumaro.plugins.open_images_format import OpenImagesConverter, OpenImagesImporter
+from datumaro.util.image import Image
+from datumaro.util.test_utils import TestDir, compare_datasets_strict
 from datumaro.components.extractor import AnnotationType, DatasetItem, Label, LabelCategories
 from datumaro.components.dataset import Dataset
 from tests.requirements import Requirements, mark_requirement
 
+class OpenImagesFormatTest(TestCase):
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_and_load(self):
+        source_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(id='a', subset='train',
+                    annotations=[Label(0, attributes={'score': 0.7})]
+                ),
+                DatasetItem(id='b', subset='train', image=np.zeros((8, 8, 3)),
+                    annotations=[Label(1), Label(2, attributes={'score': 0})]
+                ),
+            ],
+            categories={
+                AnnotationType.label: LabelCategories.from_iterable([
+                    '/m/0',
+                    ('/m/1', '/m/0'),
+                    '/m/2',
+                ]),
+            },
+        )
+
+        with TestDir() as test_dir:
+            OpenImagesConverter.convert(source_dataset, test_dir,
+                save_images=True)
+
+            parsed_dataset = Dataset.import_from(test_dir, 'open_images')
+
+            # the converter assumes that labels without a score have a score of 100%
+            source_dataset.get('b', subset='train').annotations[0].attributes['score'] = 1
```

Comment:
Consider specifying a `target/expected_dataset` prior the test body.

Reply:
I'm not sure what you mean. Do you want me to recreate the dataset again (but with the score set)? Or is there a way to copy the source dataset?

Reply:
I meant a modified copy. You can [make a COW shallow] copy of a dataset by 
```python
target_dataset = Dataset.from_extractors(source_dataset)
```
but you probably just want to create a new dataset.

Reply:
Creating a new dataset from scratch would be overly verbose and obscure the difference between the source and expected datasets. I implemented a middle-ground solution - copying the dataset and then replacing the `b` item.